src/hotspot/cpu/x86/x86.ad

```diff
@@ -8879,6 +8971,50 @@ instruct vdiv_mem_masked(vec dst, memory src2, kReg mask) %{
   ins_pipe( pipe_slow );
 %}
 
+instruct vrol_imm_masked(vec dst, immI8 shift, kReg mask) %{
+  match(Set dst (RotateLeftV (Binary dst shift) mask));
+  match(Set dst (RotateRightV (Binary dst shift) mask));
+  format %{ "vprotate_imm_masked $dst, $dst, $shift\t! rotate masked operation" %}
+  ins_encode %{
+    int vlen_enc = vector_length_encoding(this);
+    BasicType bt = Matcher::vector_element_basic_type(this);
+    int opc = this->ideal_Opcode();
+    __ evmasked_op(opc, bt, $mask$$KRegister, $dst$$XMMRegister,
+                   $dst$$XMMRegister, $shift$$constant, true, vlen_enc);
+  %}
+  ins_pipe( pipe_slow );
+%}
+
+instruct vrol_reg_masked(vec dst, vec src2, kReg mask) %{
+  match(Set dst (RotateLeftV (Binary dst src2) mask));
+  match(Set dst (RotateRightV (Binary dst src2) mask));
+  format %{ "vrotate_masked $dst, $dst, $src2\t! rotate masked operation" %}
+  ins_encode %{
+    int vlen_enc = vector_length_encoding(this);
+    BasicType bt = Matcher::vector_element_basic_type(this);
+    int opc = this->ideal_Opcode();
+    bool is_varshift = !VectorNode::is_vshift_cnt_opcode(in(2)->isa_Mach()->ideal_Opcode());
+    __ evmasked_op(opc, bt, $mask$$KRegister, $dst$$XMMRegister,
+                   $dst$$XMMRegister, $src2$$XMMRegister, true, vlen_enc, is_varshift);
```

Comment:
x86.ad is sending is_varshift for rotates but is_varshift is not passed to evrold/evrord in evmasked_op.

Reply:
Its not needed, unlike vector shifts, rotate always consume shift either as an immediate value or though a vector of same shape as source vector. Removing it.